### PR TITLE
JS exercise 6 (Lesson 23)

### PR DESCRIPTION
**Pull request by rohan-repos**

### CODE REVIEW

File: JS_exercises/exercise-6/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let currentHour = prompt("Enter the current hour(24-hour time format");
```
Comment (chriztie) [inline JS_exercises/exercise-6/index.js:1]: It's best to get texts converted to numbers before any comparisons

File: JS_exercises/exercise-6/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+if(currentHour<0 || currentHour>23){
+    alert("Incorrect value for hour. Please enter a value between 0-23");
+}
+else if(currentHour<8 || currentHour>=18)
+{
+    alert("The hour entered is outside of office hours");
+}
+else if(currentHour>=8 && currentHour<18){
```
Comment (chriztie) [inline JS_exercises/exercise-6/index.js:9]: Do you really need this condition? A simple else should do

**Review by chriztie — APPROVED** [review]

File: JS_exercises/exercise-6/index.js (hunk 2 above)
Comment (rohan-repos) [reply JS_exercises/exercise-6/index.js]: changed it to else

File: JS_exercises/exercise-6/index.js (hunk 1 above)
Comment (rohan-repos) [reply JS_exercises/exercise-6/index.js]: converted it to number

**Pull request merged**